### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by crystall1nedev**
Opening for @techpro-services, this PR fixes small bugs around documentation.

### CODE REVIEW

File: docs/PATCHEXPLAIN.md [hunk 1]
```diff
@@ -318,6 +318,103 @@ Unfortunately certain on-disk patches are required to achieve full functionality
...
+    * Avoids `addMemoryRegion/removeMemoryRegion` calls
+      * At Offset `0x5527` and `0x77993`, replace following bytes with `0x909090909090`
+  * GeForceTeslaGLDriver.bundle
+  * GeForceTeslaVADriver.bundle
+  * NVDANV50HalTesla.kext
+  * NVDAResmanTesla.kext
+    * 0x1ea59a - 0x1ea5b3: nop
+    * Replace VSLGestalt to IOLockLock or any other known symbol of the same length.
+
+* NVIDIA Web Drivers Binaries
```
Comment (Jazzzny) [inline docs/PATCHEXPLAIN.md:347]: Please list webdriver bundles for consistency

File: docs/PATCHEXPLAIN.md [hunk 2]
```diff
@@ -318,6 +318,103 @@ Unfortunately certain on-disk patches are required to achieve full functionality
...
+  * AMDFramebuffer.kext
+  * AMDLegacyFramebuffer.kext
+  * AMDLegacySupport.kext
+  * AMDRadeonVADriver.bundle
+  * AMDRadeonVADriver2.bundle
+  * AMDRadeonX3000.kext
+  * AMDRadeonX3000GLDriver.bundle
+  * AMDShared.bundle
+  * AMDSupport.kext
+  * ATIRadeonX2000.kext
```
Comment (Jazzzny) [inline docs/PATCHEXPLAIN.md:377]: Add note for AMD issue resolution

File: docs/TROUBLESHOOTING.md [hunk 3]
```diff
@@ -66,14 +66,10 @@ Then revert the snapshot
 ```sh
 bless --mount "/Volumes/Macintosh HD" --bootefi --last-sealed-snapshot
 ```
-After that, type the following
+After that, remove problematic kexts
 ```sh
-cd "/Volumes/Macintosh HD/Library/Extensions"
+find “/Volumes/Macintosh HD/Library/Extensions” -name “*.kext” -not -name “HighPoint*” -exec rm -rf {} \;
```
Comment (Jazzzny) [inline docs/TROUBLESHOOTING.md:71]: SoftRAID should also be kept

**Review by Jazzzny — CHANGES_REQUESTED** [review]

**Comment by crystall1nedev** [comment]
Was told by the original owner of the repo that this PR should be closed.

**Pull request closed without merging**